### ISSUE TITLE
[QUESTION] Obok DeDRM claims success after multiple keys, but its gibberish

### ISSUE THREAD

**Issue by hungsu** [apprenticeharper/DeDRM_tools#1793]
## CheckList

- [x] `The Title` and The `Log Title` are setted correctly.
- [x] Clarified about `my environment`.
- [x] Code block is used for `the log`.

---

## OBok tries a few keys and claims success
Seems like it thinks gibberish is valid, but it still has more keys to try

## Log
<details><summary>Log Title</summary>

```log
calibre Debug log
calibre 5.24 [64bit]  embedded-python: True is64bit: True
Windows-10-10.0.19041 Windows ('64bit', 'WindowsPE')
('Windows', '10', '10.0.19041')
Python 3.8.5
Windows: ('10', '10.0.19041', '', 'Multiprocessor Free')
Interface language: None
Successfully initialized third party plugins: Obok DeDRM (7, 2, 1)
calibre 5.24 [64bit]  embedded-python: True is64bit: True
Windows-10-10.0.19041 Windows ('64bit', 'WindowsPE')
('Windows', '10', '10.0.19041')
Python 3.8.5
Windows: ('10', '10.0.19041', '', 'Multiprocessor Free')
Interface language: None
Successfully initialized third party plugins: Obok DeDRM (7, 2, 1)
Turning on automatic hidpi scaling
devicePixelRatio: 1.0
logicalDpi: 120.0 x 120.0
physicalDpi: 169.33333333333334 x 169.33333333333334
Using calibre Qt style: True
[0.00] Starting up...
[0.01] Showing splash screen...
[0.55] splash screen shown
[0.55] Initializing db...
[0.55] db initialized
[0.55] Constructing main UI...
DEBUG:    0.0 obok::utilities.py - loading translations
DEBUG:    0.0 obok::dialogs.py - loading translations
DEBUG:    0.0 obok::config.py - loading translations
DEBUG:    0.0 obok::action_err.py - loading translations
[1.25] main UI initialized...
[1.25] Hiding splash screen
Starting QuickView
Traceback (most recent call last):
  File "calibre\db\cache.py", line 1441, in set_metadata
  File "calibre\db\cache.py", line 1364, in set_cover
  File "calibre\db\backend.py", line 1532, in set_cover
  File "calibre\utils\img.py", line 288, in save_cover_data_to
  File "calibre\utils\img.py", line 196, in image_to_data
ValueError: Failed to export image as JPEG with error: Image is empty
[50.61] splash screen hidden
[50.61] Started up in 50.61 seconds with 2 books
Running Obok DeDRM v7.2.1
DEBUG:   52.0 didn't find device
Obok v4.0.0
Copyright © 2012-2020 Physisticated et al.
C:\Users\hi\AppData\Local\Temp\tmpj8dkkmr8
DEBUG:   52.1 got kobodir C:\Users\hi\AppData\Local\Kobo\Kobo Desktop Edition
Found 16 possible keys to try.
DEBUG:   57.5 BookListTableWidget:get_books - book_num: 0
DEBUG:   57.5 BookListTableWidget:get_books - book: Caste (Oprah's Book Club)
DEBUG:   57.5 InterfacePluginAction::launchObok - number of books to decrypt: 1
Obok DeDRM v7.2.1 - Decrypting Caste (Oprah's Book Club)
Trying key:  b'01f3a613d64e85ca21d360a1e5928bd9'
Checking text:b'xP-\xc0\xcfV\xa22\xa9\x1e':
Perhaps utf-8 without BOM
Bad character at 3, value 192
Decryption failed, trying next key.
Trying key:  b'1d670202fe45d94b6d332e7753c9f0ee'
Checking text:b'\xef\xbf\xbc\x15\x01\x97^\x96\x9ba':
Perhaps utf-8 without BOM
Bad character at 0, value 239
Decryption failed, trying next key.
Trying key:  b'e28fb9deb746026e46f703f983aa1ea2'
Checking text:b'\xa3\xdf\x9d=\xc0\x06\xabq\xf8P':
Perhaps utf-8 without BOM
Bad character at 0, value 163
Decryption failed, trying next key.
Trying key:  b'32e5cc82150071daa2c75b02848c556c'
Checking text:b',:\xc8\xafV\x1eH\x10\xdc\xfc':
Perhaps utf-8 without BOM
Bad character at 2, value 200
Decryption failed, trying next key.
Trying key:  b'03f11bdd5181ebfc2dfe271ee6c77d85'
Checking text:b'\x88$<R\xd9\xd8\xaa\x94Xs':
Perhaps utf-8 without BOM
Bad character at 0, value 136
Decryption failed, trying next key.
Trying key:  b'64e3a9ce9ff7011c5ea12db99631acf9'
Checking text:b'\x84\x16Z\x92\xe4\x11!u\x1c\x1d':
Perhaps utf-8 without BOM
Bad character at 0, value 132
Decryption failed, trying next key.
Trying key:  b'f295cdd16c12760de4b7a630a510d771'
Checking text:b'5\xd8 \x0e\x9aP\xeb\xe3\xff\xc9':
Perhaps utf-8 without BOM
Bad character at 1, value 216
Decryption failed, trying next key.
Trying key:  b'e68dcc26b8218ac35c24021e2e0ffe52'
Checking text:b'#k`f<2\x1e\xc9\xa9\xac':
Perhaps utf-8 without BOM
Seems to be good text
Success!
DEBUG:   57.9 DecryptAddProgressDialog::do_book_action - decrypted book: 'Caste (Oprah's Book Club)'
Obok DeDRM v7.2.1 - Added Caste (Oprah's Book Club)
Obok DeDRM v7.2.1 - wrapping up results.
```
</details>

**Comment by ElleKayEm** [comment]
There is discussion of problems with Obok and a couple versions with fixes that ableeker has attached to issue #1718.

**Comment by hungsu** [comment]
Thank you! Sounds like I can close this then
